src/components/loader-star/loader-star.component.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import tagComponent, {
+  TagProps,
+} from "../../__internal__/utils/helpers/tags/tags";
+import { StyledLoaderStarWrapper, StyledLabel } from "./loader-star.style";
+import useLocale from "../../hooks/__internal__/useLocale";
+import useMediaQuery from "../../hooks/useMediaQuery";
+import Star from "./internal/star.component";
+import Typography from "../typography";
+
+export interface LoaderStarProps extends Omit<TagProps, "data-component"> {
+  /**
+   * The loaderStarLabel prop allows a specific label to be set.
+   * This label will be present if the user has `reduce-motion` enabled and will also be available to assistive technologies.
+   * By default the label will be `Loading...`.
+   */
+  loaderStarLabel?: string;
+}
+const LoaderStar = ({
+  loaderStarLabel,
+  ...rest
+}: LoaderStarProps): JSX.Element => {
+  const locale = useLocale();
+
+  const reduceMotion = !useMediaQuery(
+    "screen and (prefers-reduced-motion: no-preference)"
+  );
+
+  const label = (
+    <StyledLabel data-role="visible-label" variant="span" fontWeight="500">
+      {loaderStarLabel || locale.loaderStar.loading()}
+    </StyledLabel>
+  );
+
+  return (
+    <StyledLoaderStarWrapper
+      role="status"
+      {...tagComponent("loader-star", rest)}
+    >
```

Comment:
Good suggestion! Since the container has a role of "status", it [should function as an ARIA live region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#roles_with_implicit_live_region_attributes), meaning a screen reader should announce the label when possible. It's definitely worth testing this during QA, and we can revisit the `aria-labelledby` option if it doesn't work as expected :)